infra/base-images/base-builder/jcc/jcc.go

```diff
@@ -174,8 +213,17 @@ func compile(bin string, args []string) (int, string, string) {
 	return cmd.ProcessState.ExitCode(), outb.String(), errb.String()
 }
 
+func Compile(bin string, args []string) (int, string, string) {
+	// Generate ASTs f we define this ENV var.
+	if os.Getenv("JCC_GENERATE_AST") == "1" {
```

Comment:
final nit: maybe we can encode the base output path here instead of just 1? that makes this a bit more flexible/re-usable than always writing to /out.

Reply:
Yes good point.
Working on this now.

Reply:
Done!